README.md

```diff
@@ -150,10 +151,11 @@ Follow [this document](https://cloud.google.com/dataproc/docs/concepts/component
 | 0.1.0-preview                  | ✓   | ✓   |
 | 0.2.0-preview                  | ✓   | ✓   |
 | 0.2.0                          | ✓   | ✓   |
+| 0.3.0                          | ✓   | ✓   |
 
 ## Usage
-
-The connector uses Flink’s Datastream API, and can be used in Java applications. For a Flink source, it offers two read 
+The connector could be used via Flink’s Datastream API, and can be used in Java applications. 
+It could also be used via Flink's Table API and SQL. For a Flink source, it offers two read 
```

Comment:
The connector can be used with Flink's Datastream and Table APIs in Java applications. The source offers two read modes, bounded and unbounded. The sink offers at-least-once delivery gaurantee.

Reply:
Fixed.